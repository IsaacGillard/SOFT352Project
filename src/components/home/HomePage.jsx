import React from 'react';
import { Button, Label, Table } from 'reactstrap';
import { Link } from 'react-router-dom';

import Sidebar from '../shared/Sidebar';

export default function HomePage(props) {
  const { latestAlbum } = props;
  return (
    <div className="row justify-content-center">
      <div className="col-10 col-sm-7 col-md-5 col-lg-4">
        <h1>Welcome to SoundShare!</h1>
                <p>
                  SoundShare lets you save your favourite albums and artists to your own public profile from the Discogs database.
                </p>
                <Button id="reactButton" color="primary" size="lg" tag={Link} to="/account/register">Sign Up</Button>
      </div>
    </div>
  );
}
