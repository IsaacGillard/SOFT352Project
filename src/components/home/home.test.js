import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import HomePage from './HomePage';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });

test('HomePage component should render as expected', () => {
  const component = shallow(<HomePage />)
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})
