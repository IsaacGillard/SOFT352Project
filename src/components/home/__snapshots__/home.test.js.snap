// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage component should render as expected 1`] = `
<div
  className="row justify-content-center"
>
  <div
    className="col-10 col-sm-7 col-md-5 col-lg-4"
  >
    <h1>
      Welcome to SoundShare!
    </h1>
    <p>
      SoundShare lets you save your favourite albums and artists to your own public profile from the Discogs database.
    </p>
    <Button
      color="primary"
      id="reactButton"
      size="lg"
      tag={[Function]}
      to="/account/register"
    >
      Sign Up
    </Button>
  </div>
</div>
`;
